src/DssVest.t.sol

```diff
@@ -116,7 +115,7 @@ contract DssVestTest is DSTest {
 
         hevm.warp(now + 70 days);
 
-        vest.vest(id, MAX_UINT);
+        vest.vest(id, uint256(-1));
```

Comment:
Did you make this one call my overlaoaded function on purpose? It's just different than the other.